src/Service/ServiceListenerFactory.php

```diff
@@ -96,20 +97,6 @@ class ServiceListenerFactory implements FactoryInterface
     ];
 
     /**
-     * Constructor
-     *
-     * When executed under zend-servicemanager v3, injects additional aliases
-     * to ensure backwards compatibility.
-     */
-    public function __construct()
-    {
-        $r = new ReflectionClass(ServiceLocatorInterface::class);
-        if ($r->hasMethod('build')) {
-            $this->injectV3Aliases();
```

Comment:
@Xerkus you have right. That's why I add aliases into `$defaultServiceConfig` property (above)